source/frontend/src/hooks/util/useApiRequestWrapper.ts

```diff
@@ -98,7 +96,28 @@ export const useApiRequestWrapper = <
           return;
         }
         const axiosError = e as AxiosError<IApiError>;
-        onError && onError(axiosError);
+        if (onError) {
+          onError(axiosError).catch((axiosError: AxiosError) => {
```

Comment:
This change ensures that if an explicit error handler is set, the bomb icon will not display, unless the error handler re-throws the exception.